eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -14,4 +14,5 @@ steps:
         -urls $(dir -r -i *.md) 
         -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" 
         -recursive: $false 
-        -ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
\ No newline at end of file
+        -ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
+        -checkLinkGuidance: $true
```

Comment:
Can you please make this a template parameter? I can imagine some uses will not want to enforce this guidance. Can you also create template parameters for the urls and recursive parameter as well? That will make this template a little more flexible and usable.